packages/@n8n/extension-sdk/config/tsup.mjs

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from 'tsup';
+
+export function defineN8nExtensionTsupConfig() {
+	return defineConfig({
+		entry: [
+			'src/backend/**/*.ts',
+			'!src/backend/**/*.test.ts',
+			'!src/backend/**/*.d.ts',
+			'!src/backend/__tests__**/*',
```

Comment:
This glob looks broken? Also we should be skipping `__tests__` at any level.

Reply:
Good point. Fixed!